lib/interfaces/custom-scalar.interface.ts

```diff
@@ -1,6 +1,8 @@
+import { ValueNode } from 'graphql';
+
 export interface CustomScalar<T, K> {
-  description: string;
-  parseValue(value: T): K;
-  serialize(value: K): T;
-  parseLiteral(ast): K;
+  description: string | null | undefined;
+  parseValue(value: T): K | null | undefined;
+  serialize(value: K): T | null | undefined;
```

Comment:
Perhaps we should define `Maybe` type instead? Following this implementation https://github.com/graphql/graphql-js/blob/278bde0a5cd71008452b555065f19dcd1160270a/src/type/definition.d.ts#L293-L296

Reply:
Or how about using these type definitions directly?

https://github.com/graphql/graphql-js/blob/278bde0a5cd71008452b555065f19dcd1160270a/src/type/definition.d.ts#L316-L325

Reply:
Good point! I like it